src/Components/test/testassets/BasicTestApp/Index.razor

```diff
@@ -29,7 +29,10 @@
         <option value="BasicTestApp.FocusEventComponent">Focus events</option>
         <option value="BasicTestApp.FormsTest.NotifyPropertyChangedValidationComponent">INotifyPropertyChanged validation</option>
         <option value="BasicTestApp.FormsTest.SimpleValidationComponent">Simple validation</option>
+        <option value="BasicTestApp.FormsTest.SimpleValidationComponentUsingExperimentalValidator">Simple validation using experimental validator</option>
         <option value="BasicTestApp.FormsTest.TypicalValidationComponent">Typical validation</option>
+        <option value="BasicTestApp.FormsTest.TypicalValidationComponentUsingExperimentalValidator">Typical validation using experimental validator</option>
+        <option value="BasicTestApp.FormsTest.ExperimentalValidationComponent">Experimental validation</option>
         <option value="BasicTestApp.GlobalizationBindCases">Globalization Bind Cases</option>
```

Comment:
Wouldn't it be better to duplicate the tests for the experimental validator completely instead of doing `if...else` in some of the infrastructure if the goal here is that for 5.0 this becomes the de-facto framework behavior? That way then its a matter of removing code.

Reply:
I thought it'd be less work than maintaining two copies of these. Do you feel strongly about this?